### ISSUE TITLE
finetune unet problem?

### ISSUE THREAD

**Issue by SophieZhou** [esdolo/FreeVS#12]
Hi, Do you fine tune unet fully or just parts of some blocks? When I tune unet fully training SVD, the loss maybe nan easily. I am not sure with lr=1e-4, the unet can be finetuned fully.

**Comment by esdolo** [comment]
I train the full unet with lr=5e-5, AdamW optimizer, and warm-up steps = 500. The loss is stable under a batch size = 8.

**Comment by SophieZhou** [comment]
> I train the full unet with lr=5e-5, AdamW optimizer, and warm-up steps = 500. The loss is stable under a batch size = 8.

Thanks very much.

**Comment by SophieZhou** [comment]
> I train the full unet with lr=5e-5, AdamW optimizer, and warm-up steps = 500. The loss is stable under a batch size = 8.

Then, the convnext model with learning rate 1e-4?
